### PR TITLE
Update migration-migrate-settings.md

### PR DESCRIPTION
**Pull request by TheGamesmenDev**
Note that Magento needs to be deployed in Default Mode (or at least, NOT Developer Mode) for the Data Migration Tool to work without XML validation errors.

## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will update Magento 2.1 and greater documentation regarding the Data Migration Tool

Magento 2.1 and greater documentation regarding the Data Migration Tool
## Additional information

List all affected URLs 
https://devdocs.magento.com/guides/v2.3/migration/migration-migrate-settings.html
- ...
- ...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3771) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3771) before we can accept your contribution.<br/>

**Comment by magento-engcom-team** [comment]
@TheGamesmenDev thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/migration/migration-migrate-settings.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ According to our data migration [order]({{ page.baseurl }}/migration/migration-m
 
 2. Change to the Magento `/bin` directory or make sure it is added to your system PATH.
 
+3. Make sure Magento is deployed in Default Mode (Developer Mode will cause validation errors in the Migration Tool)
```
Comment (jeff-matthews) [inline guides/v2.1/migration/migration-migrate-settings.md]:
```suggestion
{: .bs-callout .bs-callout-info }
Make sure Magento is deployed in default mode. Developer mode causes validation errors in the migration tool.
```

**Comment by dobooth** [comment]
Closing since 1) v2.1 file. 2) Haven't heard from The Gamesman since this, his first, PR. 3) Easy enough to make a new PR with this small fix.

**Pull request closed without merging**